src/js/index.js

```diff
@@ -15,10 +15,18 @@ function onIndexLoad() {
 /* eslint-enable no-unused-vars */
 
 // INDEX PAGE FUNCTIONS
-
+function removeRadioButtons() {
+  var buttons = document.getElementsByClassName('btn-label');
+  for (var a = 0; a < buttons.length; a++) {
+    if (buttons[a].firstChild.value == 'openjdk9' || buttons[a].firstChild.value == 'openjdk10') {
+      buttons[a].style.display = 'none';
```

Comment:
The reason these are hidden is because they are not 'current' releases, i.e. no longer in support at the upstream source projects.
I wonder if it is possible to use an attribute of the builds to determine that when creating the buttons, rather than hard-code the exceptions here?

Reply:
We could add that to the API - I'll open an issue over there

Reply:
https://github.com/AdoptOpenJDK/openjdk-api/issues/118

Reply:
My latest commit (where the LTS flag is added) actually adds the functionality for us to do this, as we store whether this version is in LTS in the `config.json`. 

So yes it should be possible, I'll push up a proposed fix for you to take a look at. 

Thanks.

Reply:
Woops... I didn't see your comments @karianna, Would you prefer to leave it in that issue?

Reply:
I don't have the capability to do it on creation of the buttons yet. I did try that approach but would have needed to refactor various parts of the project. 

I'm about to push up a new commit that will at least remove the `hard coded` version names and will at least future proof against newer JDKs being added in the mean time.